DuckDuckGo/SecureVault/View/PasswordManagementViewController.swift

```diff
@@ -201,47 +199,20 @@ final class PasswordManagementViewController: NSViewController {
             .store(in: &cancellables)
     }
 
-    private func setUpEmptyStateMessageAttributedText() {
+    private func setUpEmptyStateMessageView() {
         guard let listModel else { return }
-        emptyStateMessage.delegate = self
 
-        let linkAttributes: [NSAttributedString.Key: Any] = [
-            .foregroundColor: NSColor.linkBlue,
-            .cursor: NSCursor.pointingHand
-        ]
+        let message = " \(listModel.emptyStateMessageDescription) [\(listModel.emptyStateMessageLinkText)](\(listModel.emptyStateMessageLinkURL))"
 
-        emptyStateMessage.linkTextAttributes = linkAttributes
-
-        let attachment = NSTextAttachment()
-        attachment.image = NSImage(resource: .lockSolid16).tinted(with: NSColor.blackWhite80)
-        attachment.bounds = CGRect(x: 0, y: -1, width: 12, height: 12)
-        let attributedTextImage = NSMutableAttributedString(attachment: attachment)
-
-        let string = NSMutableAttributedString(attributedString: attributedTextImage)
-
-        let messageString = NSMutableAttributedString(string: " " + listModel.emptyStateMessageDescription + " ")
-        string.append(messageString)
-
-        let linkString = NSMutableAttributedString(string: listModel.emptyStateMessageLinkText, attributes: [
-            .link: listModel.emptyStateMessageLinkURL
-        ])
-        string.append(linkString)
-
-        let paragraphStyle = NSMutableParagraphStyle()
-        paragraphStyle.alignment = .center
-        string.addAttributes([
-            .cursor: NSCursor.arrow,
-            .paragraphStyle: paragraphStyle,
-            .font: NSFont.systemFont(ofSize: 13, weight: .regular),
-            .foregroundColor: NSColor.blackWhite80
-        ], range: NSRange(location: 0, length: string.length))
-
-        let maxSize = NSSize(width: 280, height: 20000)
-        let bounds = string.boundingRect(with: maxSize, options: .usesLineFragmentOrigin)
-
-        emptyStateMessageHeight.constant = bounds.height
-
-        emptyStateMessage.textStorage?.setAttributedString(string)
+        let hostingView = NSHostingView(rootView: PasswordManagementEmptyStateMessage(
+            message: message,
+            image: .lockSolid16
+        ))
+        hostingView.frame = emptyStateMessageContainer.bounds
+        hostingView.autoresizingMask = [.height, .width]
+        hostingView.translatesAutoresizingMaskIntoConstraints = true
+        emptyStateMessageContainer.addSubview(hostingView)
+        emptyStateMessageHeight.constant = listModel.emptyStateMessageViewHeight
```

Comment:
I think the height constant is causing some issues with other languages which have longer strings 
<img width="480" alt="image" src="https://github.com/user-attachments/assets/fbc5e802-ca38-4470-b45d-f64daa133eab">
<img width="480" alt="image" src="https://github.com/user-attachments/assets/ae31b702-dfe0-47b7-a54a-69159dd3e4b6">